### PR TITLE
French localisation

### PR DESCRIPTION
**Pull request by LordaeronEmbassy**
## Changelog:
- Everything except titles, provinces, and character names translated to the French language.

### CODE REVIEW

File: localisation/00_memewc_united_localisation.csv [hunk 1]
```diff
@@ -1,111 +1,111 @@
...
+RULE_ARTIFACT_SPAWN_X1_5_DESC;Amount of generic artifacts spawned on gamestart is modified by �G1.5�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G1.5�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �G1,5�! erh�ht.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X2_DESC;Amount of generic artifacts spawned on gamestart is modified by �G2�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G2�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �G2�! erh�ht.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X0_5_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.5�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G0.5�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,5�! reduziert.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X0_25_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.25�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G0.25�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,25�! reduziert.;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_memewc_united_localisation.csv:8]:
```suggestion
RULE_ARTIFACT_SPAWN_X0_5_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.5�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �R0.5�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,5�! reduziert.;;;;;;;;;;;x
RULE_ARTIFACT_SPAWN_X0_25_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.25�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �R0.25�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,25�! reduziert.;;;;;;;;;;;x
```

File: localisation/00_wc_mnm_replace.csv [hunk 2]
```diff
@@ -2,6 +2,5 @@
...
+devil_worshipers_dark_healing_power_has_mystics_dlc_true_key;Power: �YDark Healing�!\nEnables �Gright-clicking�! on your character's portrait in order to perform a dark ritual to heal yourself, in exchange for �Y[Root.Society.GetCurrency]�!.;Pouvoir : �YGu�rison d�moniaque�!\nPermet d'ex�cuter un rituel mal�fique pour vous soigner, en �change de �Y[Root.Society.GetCurrency]�!.;Macht: �YDunkle Heilung�!\nErm�glicht das Rechtsklicken auf das Portr�t Eures Charakters, um f�r �Y[Root.Society.GetCurrency]�! ein dunkles Ritual zu Eurer eigenen Heilung durchzuf�hren.;;Poder: �YCuraci�n oscura�!\nActiva el clic con el bot�n secundario en el retrato de tu personaje para llevar a cabo un ritual oscuro para curarte a cambio de �Y[Root.Society.GetCurrency]�!.;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_wc_mnm_replace.csv:6]: Are you sure about this?
It's vanilla loc.

Comment (arithon) [reply localisation/00_wc_mnm_replace.csv:6]: are we making replacement files for the individual vanilla files now?

Comment (MemeWizard) [reply localisation/00_wc_mnm_replace.csv:6]: Well, IIRC, it was created before current replacement file.
Also, this is not replacement. It's mod lines with vanilla localization.

File: localisation/00_memewc_united_localisation.csv (hunk 1 above)
Comment (zumbak04) [reply localisation/00_memewc_united_localisation.csv]: Done in 60f0f17513eb8ea4e52b163acd0cf6c28eeae598.

**Pull request merged**